manager/test/api/src/test/resources/test-plans/searching-testPlan.xml

```diff
@@ -1,46 +1,69 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Line endings?

Reply:
I'll fix that. It came from the original commit I stacked on. BTW, here's where you can set it to auto-detect what line-endings to use, which removes all the pain in my experience!
![image](https://user-images.githubusercontent.com/423513/110147528-b305b800-7dd3-11eb-8296-67fd3f1a713e.png)